DietPi-Docs | Restic: is a fast and secure backup program (#880)
MichaIng/DietPi-Docs: docs/software/cloud.md

***

> Official documentation: <https://rclone.org/>
> Source code: <https://github.com/rclone/rclone>

Reviewer: The spaces need to stay here, otherwise there will be no line breaks:
```suggestion
Official documentation: <https://rclone.org/>  
Source code: <https://github.com/rclone/rclone>  
```

License: [MIT](https://github.com/rclone/rclone/blob/master/COPYING)

## Restic

Restic is a fast and secure backup program.

![Restic logo](../assets/images/restic-logo.png){: width="300" height="69" loading="lazy"}

=== "Quick start"

> In order to create a repository at `/srv/restic-repo`, run the following command and enter the same password twice:

Reviewer: I suggest to use the DietPi userdata path to align with other software options on DietPi:
```suggestion
In order to create a repository at `/mnt/dietpi_userdata/restic-repo`, run the following command and enter the same password twice:
```

```sh

> $ restic init --repo /srv/restic-repo

Reviewer: ```suggestion
$ restic init --repo /mnt/dietpi_userdata/restic-repo
```

> In order to create a repository at `/srv/restic-repo`, run the following command and enter the same password twice:
> 
> ```sh
> $ restic init --repo /srv/restic-repo
> enter password for new repository:
> enter password again:
> created restic repository 085b3c76b9 at /srv/restic-repo

Reviewer: I suggest to use the DietPi userdata path to align with other software options on DietPi:
````suggestion
In order to create a repository at `/mnt/dietpi_userdata/restic-repo`, run the following command and enter the same password twice:

```sh
$ restic init --repo /mnt/dietpi_userdata/restic-repo
enter password for new repository:
enter password again:
created restic repository 085b3c76b9 at /mnt/dietpi_userdata/restic-repo
````

Please note that knowledge of your password is required to access the repository.
Losing your password means that your data is irrecoverably lost.
```

!!! warning "Remembering your password is important! If you lose it, you won’t be able to access data stored in the repository."

For more information please read: [Preparing a new repository](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html)

***

> Official documentation: <https://restic.readthedocs.io>
> Official website: <https://restic.net>
> Source code: <https://github.com/restic/restic>

Reviewer: Adding linebreaks:
```suggestion
Official documentation: <https://restic.readthedocs.io>  
Official website: <https://restic.net>  
Source code: <https://github.com/restic/restic>  
```

Source code: <https://github.com/rclone/rclone>  
License: [MIT](https://github.com/rclone/rclone/blob/master/COPYING)

## Restic

Restic is a fast and secure backup program.

> ![Restic logo](../assets/images/restic-logo.png){: width="300" height="69" loading="lazy"}

Reviewer: ```suggestion
![Restic logo](../assets/images/restic-logo.png){: width="250" height="250" loading="lazy"}
```
